components/dropdown/user/src/index.js

```diff
@@ -32,12 +32,18 @@ class DropdownUser extends Component {
     })
   }
 
-  _renderLink = ({text, url, icon: Icon, notifications}, index) => {
+  _renderLink = ({text, url, icon: Icon, notifications, highlight}, index) => {
     const Link = this.props.linkFactory
+    const linkClassName = cx('sui-DropdownUserMenu-listLink', {
+      'sui-DropdownUserMenu-listLinkHighlight': highlight
```

Comment:
I know it's already merged, but you know me, and I like to review everything 😜. In this case, you should have use a state class, not a modifier. So that it should be something like this:
```
const linkClassName = cx('sui-DropdownUserMenu-listLink', {
       'is-highlighted': highlight
})
```